pkgs/desktops/pantheon/apps/elementary-music/default.nix

```diff
@@ -66,34 +60,25 @@ stdenv.mkDerivation rec {
     wrapGAppsHook
   ];
 
-  buildInputs = with gst_all_1; [
-    clutter-gtk
+  buildInputs = [
     elementary-icon-theme
+    glib
     granite
-    gst-plugins-bad
-    gst-plugins-base
-    gst-plugins-good
-    gst-plugins-ugly
-    gstreamer
     gtk3
-    json-glib
-    libaccounts-glib
-    libdbusmenu
     libgda
     libgee
     libgpod
     libhandy
-    libnotify
     libpeas
-    libsignon-glib
-    libsoup
     taglib
     zeitgeist
-  ];
-
-  mesonFlags = [
-    "-Dplugins=audioplayer,cdrom,ipod"
-  ];
```

Comment:
This is the default value on 5.1.1 and on 7.0 we no longer have this meson option.